libcudacxx/include/cuda/std/detail/libcxx/include/__cuda/ptx/ptx_isa_target_macros.h

```diff
@@ -0,0 +1,63 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+
+#ifndef _CUDA_PTX_PTX_ISA_TARGET_MACROS_H_
+#define _CUDA_PTX_PTX_ISA_TARGET_MACROS_H_
+
+#include "../../../../../../../nv/target" // __CUDA_MINIMUM_ARCH__ and friends
```

Comment:
Ok this is ridiculous. In the config we go with `#include <nv/target>` which should be fine here too

Reply:
Done.